src/utils/session.ts

```diff
@@ -1,6 +1,6 @@
 import { Request } from "express";
 import { isDetails } from "private-api-sdk-node/dist/services/presenter-account/types";
-import { type Details } from "private-api-sdk-node/src/services/presenter-account/types";
+import { Name, type Details } from "private-api-sdk-node/src/services/presenter-account/types";
```

Comment:
Since this is just importing the Name type it should be a type import just like how `Details` was imported.